src/machines/modelingMachine.ts

```diff
@@ -748,14 +748,6 @@ export const modelingMachine = setup({
         })
       })().catch(reportRejection)
     },
-    'conditionally equip line tool': ({ event: { type } }) => {
-      if (type === 'xstate.done.actor.animate-to-face') {
-        sceneInfra.modelingSend({
-          type: 'change tool',
-          data: { tool: 'line' },
-        })
-      }
-    },
```

Comment:
Doesn't this remove the line tool being auto-equiped on sketch on face?

Reply:
Let me check, I paired with Kurt when he explained this logic and we determined we could delete it. I'll try if this breaks that workflow.

One problem is, the xstate machine is wrong when entering sketch mode from a blank KCL file. It triggers enabling the line tool twice which is wrong.

Reply:
![image](https://github.com/user-attachments/assets/3e97e0de-7d7d-4f59-8dca-5486db1758f8)

Just tested this workflow and I was able to sketch on face and it auto selected the line tool.